content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -144,9 +144,66 @@ For more information on version skews, see:
 * Kubernetes [version and version-skew policy](/docs/setup/release/version-skew-policy/)
 * Kubeadm-specific [version skew policy](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#version-skew-policy)
 
+{{< warning >}}
+Kubernetes has two different package repositories starting from August 2023.
+The Google-hosted repository is considered deprecated and we will stop
+publishing packages to that repository starting with Kubernetes 1.30.0.
```

Comment:
Is this an absolute commitment, or just likely? Only say when we'll stop if we're 100% committed.

Reply:
@sftim This is something to verify with leads but most of them are not available this week. I'll make sure to follow up on this early next week.

Reply:
I removed the timeline but still left a note that this will happen in the future.

Reply:
We decided that Kubernetes 1.29 is going to be cutover. https://kubernetes.slack.com/archives/CCK68P2Q2/p1691085627058939